client/my-sites/activity/activity-log-item/style.scss

```diff
@@ -234,6 +235,21 @@
 		}
 	}
 
+	.server-actor-icon {
+		/* stylelint-disable-next-line declaration-property-unit-allowed-list */
```

Comment:
Not in this PR but `50%` should really just be on the list of allowed radii.

Reply:
Agree https://github.com/Automattic/wp-calypso/pull/71379